scripts/Client.js

```diff
@@ -23,7 +24,7 @@ function onPageAttach(page) {
 
 
 
-function getRow(row) {
+function getRow(row, campaign) {
```

Comment:
I see that you use 'campaign' as a parameter a lot. I suggest making it a global variable so you wouldn't have to pass it client functions but only to server's when necessary

Reply:
Will do!